flixel/graphics/frames/FlxBitmapFont.hx

```diff
@@ -660,4 +660,15 @@ class FlxBitmapFont extends FlxFramesCollection
 		font.updateSourceHeight();
 		return font;
 	}
+	
+	/**
+	 * Checks if the specified code is one of the Unicode Combining Diacritical Marks
+	 * @param	Code	The charactercode we want to check
+	 * @return 	Bool	Returns true if the code is a Unicode Combining Diacritical Mark
+	 */
+	public static function isUnicodeComboMark(Code:Int):Bool 
```

Comment:
Not quite sure why you put this here. Do you expect this to be useful on its own as an API that people might use directly? If so, `FlxBitmapFont` is a bit of an awkward place for it, `FlxStringUtil` might be a better fit. If not, I'd just move it to `FlxBitmapText` (the only place it's used) and keep it private for now.

Reply:
I guess I was thinking that it belonged within the font itself... less than the BitmapText object...
I can move it.